docs/src/app/components/Divider/README.md

```diff
@@ -0,0 +1,5 @@
+## Divider
+
+Dividers group and separate content within lists and page layouts. The divider is a thin rule, lightweight yet sufficient to distinguish content visually and spatially.
```

Comment:
Could you attach a link to https://www.google.com/design/spec/components/dividers.html to the `Dividers` word?